tests/python/pytorch/graphbolt/impl/test_fused_csc_sampling_graph.py

```diff
@@ -973,8 +973,15 @@ def test_sample_neighbors_homo(labor, indptr_dtype, indices_dtype):
 
     # Generate subgraph via sample neighbors.
     nodes = torch.tensor([1, 3, 4], dtype=indices_dtype)
+    nodes_in_incorrect_dtype = torch.tensor(
+        [1, 3, 4],
+        dtype=(torch.int64 if indices_dtype == torch.int32 else torch.int32),
+    )
+    fanouts = torch.LongTensor([2])
     sampler = graph.sample_layer_neighbors if labor else graph.sample_neighbors
-    subgraph = sampler(nodes, fanouts=torch.LongTensor([2]))
+    with pytest.raises(AssertionError):
```

Comment:
add the `match` part to verify the assertion details like this: https://github.com/dmlc/dgl/blob/1db71cfefec88e05299c6d29464f7e1737adb4b9/tests/python/pytorch/graphbolt/test_itemset.py#L27-L31